Automattic/crowdsignal-plugin
# Add command for creating a demo post

This PR adds a `create-block-poll` entry on management page switch  to trigger a post create with blocks.

#### Changes proposed in this Pull Request:

* Add a new action entry on `management_page_load`: `create-poll-block`
* Browsing to the action will create a post and insert pre-set content in it: intro and demo for each of our blocks

#### Testing instructions:

* Checkout on your docker environment
* Make sure the plugin is properly configured to connect to Crowdsignal, visit `/wp-admin/admin.php?page=polls`, you should see the dashboard with your content in it
* Visit `/wp-admin/admin.php?page=polls&action=create-block-poll`. After a couple of seconds you should be redirected to the editor with a newly created post with all the pre-set content
* Visit your Crowdsignal dashboard to see the newly created items
